src/email.js

```diff
@@ -0,0 +1,41 @@
+import debug from 'debug';
+import nodemailer from 'nodemailer';
+import EmailError from './errors';
+
+const log = debug('uwave:api:v1:email');
+
+export function sendEmail(emailAddress, token, options) {
+  const smtpOptions = {
+    host: 'localhost',
+    port: 25,
+    debug: true,
+    tls: {
+      rejectUnauthorized: false,
+    },
+  };
+
+  var mailTransport = null;
+  if (options.mailTransport) {
+    mailTransport = options.mailTransport;
+  } else {
+    mailTransport = nodemailer;
+  }
+
+  // create reusable transporter object using the default SMTP transport
+  const transporter = mailTransport.createTransport(smtpOptions);
+
+  // setup e-mail data with unicode symbols
+  const mailOptions = {
+    from: options.from,
+    to: emailAddress,
+    subject: options.subject,
+    text: `reset url is: https://welovekpop.club/reset/${token}`,
+  };
+
+  // send mail with defined transport object
+  if (transporter.sendMail(mailOptions)) {
```

Comment:
`sendMail` returns a promise, you need to wait for the promise to resolve before you can use the result. Here it'll always return `true` because the `if` will cast a Promise object to `true`, even if it represents a failed operation.

Here we'd want to return the Promise object if something failed, so that the web API will return an error response. If it all succeeded we want to return the Promise object but with the contents redacted (since nodemailer returns a bunch of information about the server it sends mail to or whatever). The simplest way to do that would be to:

```js
return transporter.sendMail(mailOptions)
  .then(() => ({})) // ← confusing but () => ({}) is a function that returns an empty object
```

Which would return a successful Promise object with the success value `{}` (empty object) if the email was sent, or a failed Promise object if something went wrong.